azure-devops/projects/cgn-onboarding-portal-projects/cgn-onboarding-portal-backend.tf

```diff
@@ -0,0 +1,176 @@
+variable "cgn-onboarding-portal-backend" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "cgn-onboarding-portal-backend"
+      branch_name    = "master"
+      pipelines_path = ".devops"
+    }
+    pipeline = {
+      # TODO ?
```

Comment:
```suggestion
      # TODO ?
      prod = {
        webAppName = "cgnonboardingportal-p-portal-backend1"
      }
      uat = {
        webAppName = "cgnonboardingportal-u-portal-backend1"
      }
```

Reply:
Yes, if we need to rename the web app we don't need to change the application pipeline